app/models/wiki_content/revision.rb

```diff
@@ -71,6 +71,6 @@ def plagiarism_report_link
   def references_added
     current_refs = features['feature.wikitext.revision.ref_tags'] || 0
     prev_refs = features_previous['feature.wikitext.revision.ref_tags'] || 0
-    current_refs - prev_refs
+    self.references = current_refs - prev_refs
```

Comment:
Per my note on the migration, I think sticking with the previous implementation here is better.